Pyrado/scripts/evaluation/eval_policies_domain_grid_1d.py

```diff
@@ -58,12 +58,24 @@
         args.max_steps = 2000
         print_cbt(f"Set maximum number of time steps to {args.max_steps}", "y")
 
+    # Get the experiment's directory to load from
+    if args.env_name is None:
```

Comment:
I don't liek this if-else block. Let's stick to one of them (I think the if-part is better). remove the print statement